ghost/signup-form/src/App.tsx

```diff
@@ -29,17 +30,18 @@ const App: React.FC<AppProps> = ({scriptTag}) => {
         } as Page);
     };
 
+    const i18n = i18nLib.default('en', 'signup-form');
     const context: AppContextType = {
         page,
         api,
         options,
         setPage: _setPage,
+        i18n: i18n.t,
```

Comment:
I __think__ we'll need to keep this as `t: i18n.t` like I do in Portal, because i18next-parser extract the strings by detecting `t('...')`. It is configurable but I think it's also good to keep consistent